testing/onboarding.go

```diff
@@ -123,26 +123,7 @@ func OnboardInitialUser(
 				},
 			},
 			wants: wants{
-				errCode: errors.EEmptyValue,
-			},
-		},
-		{
-			name: "missing password should fail",
```

Comment:
Other than the `name` here, this case was identical to the case above (missing bucket). The code doesn't actually enforce that a password is included in the request body, so I deleted the case instead of fixing it up.